{{ cookiecutter.project_shortname }}/.github/workflows/tests.yml

```diff
@@ -21,35 +21,74 @@ jobs:
     runs-on: ubuntu-20.04
```

Comment:
@mvidalgarcia this is how the `tests.yml` should look like if there are services. Note that `excludes` are kind of arbitrary... a definitive list can either made through https://github.com/inveniosoftware/cookiecutter-invenio-module/pull/141#discussion_r534029729 or we can simply leave it empty and fill on a case by case basis depending on the repo.